### DISCUSSION TITLE
Plan between Thursday 06Jan to Monday 10Jan

### DISCUSSION THREAD

**Discussion by amberwnaushahi** [ALIYA2Group/Mod20_Segment_1#9] [Q&A]
Jan 6 - Go through segment deliverables, milestones and roles. Establish meeting time and duration for next few sessions.

Research on Topics

Jan 7 - 9  -  Decide on a topic for the project—Decide which technologies will be used. Think of a question that can be answered using data. Source a dataset that will suit your needs (you can even use multiple datasets if applicable). Begin to clean, organize, and perform exploratory data analysis on your datasets so that they're ready for analysis. Include mockups of a machine learning model and a database.
Jan 10 - Discuss with Sal

Should Amber Update Responsibility Matrix and Charter?

**Comment by 735713038455163** [comment]
Thank you @Amber N. @Aryam for attending our session from 8pmEST-7pmCST to 10pmEST-9pmCST! We have completed a number of items tonight including 1) the communication protocol 2) the google drive set-up 3) github orientation on discussion boards, comments, issues milestones 4) and scheduled our next meeting for tomorrow same time, same place for our idea presentations see you then! (edited) We established our roles and completed our read me. we are off to a great start.
